docs/site/src/demos/svg-icon/svg-icon.md

```diff
@@ -0,0 +1,20 @@
+# SVG Icon
```

Comment:
That's an option I guess. Slightly conflicted as to whether they'll be overlooked if not in the component demos section.